DataManagementSystem/Client/FileCatalogClientCLI.py

```diff
@@ -178,7 +179,19 @@ def __getModeString(self,perm):
       
     return pstring  
   
-  def printListing(self,reverse,timeorder):
+  def human_readable_size(self,num,suffix='B'):
```

Comment:
Just a stylistic comment: in DIRAC we use camelCase with the first letter in lower case instead of underscores_case. This is true for functions, methods and parameters.